### PR TITLE
games-fps/prboom-plus: Bump to version 2.5.1.7.82

### PR DESCRIPTION
**Pull request by vilhelmgray**
Closes: https://bugs.gentoo.org/723862

### CODE REVIEW

File: games-fps/prboom-plus/prboom-plus-2.5.1.7.82.ebuild [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+MY_PV=$(ver_rs 4 "um+git")
+DESCRIPTION="An enhanced clone of the classic first-person shooter Doom"
+HOMEPAGE="https://github.com/coelckers/prboom-plus/"
+SRC_URI="http://deb.debian.org/debian/pool/main/p/prboom-plus/${PN}_${MY_PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+ GPL-3+ BSD BSD-2 BSD-with-disclosure CC-BY-3.0 CC0-1.0 LGPL-2.1+ MIT public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dumb fluidsynth mad +net +opengl +pcre portmidi sdl2-image +sdl2-mixer server vorbis zlib"
```
Comment (chewi) [inline games-fps/prboom-plus/prboom-plus-2.5.1.7.82.ebuild:16]: Any particular reason for now enabling `net` by default?

File: games-fps/prboom-plus/prboom-plus-2.5.1.7.82.ebuild [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		-DWITH_PORTMIDI="$(usex portmidi)"
+		-DDOOMWADDIR="${EPREFIX}/usr/share/doom"
+		-DWAD_INSTALL_PATH="${EPREFIX}/usr/share/doom"
+		-DBUILD_SERVER="$(usex server)"
+	)
+	cmake_src_configure
+}
+
+src_install() {
+	doicon ICONS/${PN}.svg
```
Comment (chewi) [inline games-fps/prboom-plus/prboom-plus-2.5.1.7.82.ebuild:61]: I would prefer `doicon -s scalable ICONS/${PN}.svg` here.

File: games-fps/prboom-plus/prboom-plus-2.5.1.7.82.ebuild (hunk 1 above)
Comment (vilhelmgray) [reply games-fps/prboom-plus/prboom-plus-2.5.1.7.82.ebuild:16]: PrBoom-Plus is most popular with speedrunners, so I suppose disabling net by default shouldn't be that bad; I'm not sure how popular it is for network play. I'll go ahead and remove the default enable then.

**Comment by vilhelmgray** [comment]
> Thanks! P.S. Did you see I added Eureka and SLADE? I've been doing some mapping. grin

That's great! I was planning on making an ebuild for SLADE but it looks like you beat me too it haha.

Speaking of mapping, I've been maintaining [chocolate-doom and crispy-doom in the GURU tree](https://gitweb.gentoo.org/repo/proj/guru.git/tree/games-fps). Would it be possible to move these over to the proper **::gentoo** tree? They're useful for testing maps to verify some basic compatibility with the other more sophisticated doom engines.

**Comment by chewi** [comment]
Of course. I have slightly less interest in these more "boring" ports but I love the fact that we're quickly becoming the best distro for DOOM!

**Comment by vilhelmgray** [comment]
They're pretty simple source ports so I don't expect them to be much trouble for me to maintain. Thanks!

I've pushed the latest versions so the existing respective PRs:
https://github.com/gentoo/gentoo/pull/12011
https://github.com/gentoo/gentoo/pull/12012

**Pull request closed without merging**